src/os.cc

```diff
@@ -20,6 +20,9 @@
 
 #  ifndef _WIN32
 #    include <unistd.h>
+#    ifdef _WRS_KERNEL  //added VxWorks7 kernel support
+#        include <ioLib.h>
+#    endif
```

Comment:
I think this better be moved outside of `ifndef _WIN32` block because those are independent.
Also it would be nice to add a comment clarifying that the new include is for `getpagesize`, something like 

```c++
# include <ioLib.h>  // getpagesize
```

Reply:
Thank you for your review. 
I have changed my code.